packages/cli/src/commands/org/config/monitor.ts

```diff
@@ -100,4 +100,17 @@ export default class Monitor extends SnapshotBase {
       },
     };
   }
+
+  private async getTargetOrg() {
```

Comment:
This is repeated across many command and could be extracted/shared.

The only difference seems to be the command itself (eg: `Monitor` + a reference to the command instance `this`), which could be passed as arguments to a shared function.

Reply:
Honestly, I feel like a minimal base class just to return the target org would be a good idea for this case.
what do you think?

Reply:
> Honestly, I feel like a minimal base class just to return the target org would be a good idea for this case.
> what do you think?
> It will only be used by commands that actually connect to a target organization...
> 
> Or it can also be a util function in SnapshotCommon

I think it's better to stay in one paradigm here, the idea of Oli to have a pure function that given a specific Command, will return you the organization.

However, you could introduce specificity in the command that are allowed to call it.

i.e.
```ts
class Monitor extends Commands implements IWithTargetOrg
```